### DISCUSSION TITLE
[Documentation]: Reverse proxy guide not (fully) updated for 0.14

### DISCUSSION THREAD

**Discussion by sandnabba** [blakeblackshear/frigate#12935] [Config Support] [answered]
### Describe the problem you are having

Hi!

Just upgraded to 0.14 and had some issues along the way.

1. **TLS**: Got a HTTP 400, "The plain HTTP request was sent to HTTPS port" error after switching to the new port with authentication.  
After some Googling I found an answer here in some other ticket about setting:
```yml
tls:
  enabled: false
  ```
When using a reverse proxy. However, there is no info about this in the documentation.

2. **Authentication:** Speaking of the new port, the reverse proxy guide still mention `Restrict access to your Frigate instance at the proxy using, for example, password authentication`   
This should probably be updated to mention that Frigate now has built in authentication.
 

I could probably rewrite this myself and submit a PR. But before doing that I just want some input regarding:
* If there are some other consequences of disabling TLS?
* And if unauthenticated port 5000 will still be supported in the future? (If people wants to use their own authentication service)

I noticed in the "authentication" documentation that you could set `auth.enabled: False`, but I'm not sure if that is the same as just using port 5000?

### Version

0.14.0

### Frigate config file

```yaml
tls:
  enabled: false
```

### Relevant log output

```shell
The plain HTTP request was sent to HTTPS port
```

### Operating system

Other Linux

### Install method

Docker Compose

### Object Detector

CPU (no coral)

**Comment by NickM-27** [comment]
Updating the docs would be appreciated. 

1. no, as long as TLS is provided by the proxy
2. I don't see a reason for it to go away

**Comment by mitchplze** [comment]
I had the same issue using Caddy, and disabling TLS like this fixed it. Thanks @sandnabba. Can also help with docs if you need.

**Comment by richarddeweerd** [comment]
thank you @sandnabba you saved me from banging my head against the wall for an hour

**Comment by sandnabba** [comment, marked as answer]
So, I've created a PR: https://github.com/blakeblackshear/frigate/pull/13075

This is my first contribution to the Frigate project. Should I link this discussion to the PR in some way, or convert to an issue? Or is an "independent" PR ok? :)

Best regards

**Reply by NickM-27** [reply]
all good as is, thank you!
